Azure Sentinel DevOps Template/readme.md

```diff
@@ -0,0 +1,4 @@
+# Azure DevOps Generator
```

Comment:
How does a customer use this DevOps template? The Readme needs to cover detailed guidance.

Reply:
Added detailed steps, readme.md should be updated

Reply:
Hey Preeti, the templates covers every phase of an Azure Sentinel deployment, including, data collection. Will follow up offline.

Reply:
Thanks. As we discussed please make the following changes:
1. Make this a zip file for easier deployment
2. Detailed readme on what users will get - how to use this tool/templates - how to deploy this.
3. Please update the PR to move this folder to  https://github.com/Azure/Azure-Sentinel/tree/master/Solutions - instead of being directly under the root which it is currently.

Reply:
Observed #1 and #2 are done. I've commented on the changed code for #3 hence resolving this conversation.